tensorflow/tools/ci_build/ci_sanity.sh

```diff
@@ -321,7 +321,7 @@ do_external_licenses_check(){
   EXTRA_LICENSES_FILE="$(mktemp)_extra_licenses.log"
 
   echo "Getting external dependencies for ${BUILD_TARGET}"
- bazel query "attr('licenses', 'notice', deps(${BUILD_TARGET}))" --no_implicit_deps --no_host_deps --keep_going \
+ bazel query "attr('licenses', 'notice', deps(${BUILD_TARGET}))" --keep_going \
```

Comment:
This seems to break do_pip_package_licenses_check checks?

Reply:
resolved. There seems to be some real issues in the licence check.